src/ESUGApp-Model/ERTalk.class.st

```diff
@@ -135,6 +173,34 @@ ERTalk >> descriptionTitle [
 	^ tmp yourself
 ]
 
+{ #category : #accessing }
+ERTalk >> discardAcceptance [
+	^ accepted := false.
+]
+
+{ #category : #accessing }
+ERTalk >> displayStatusClass [
+	self accepted ifTrue: [ ^ 'hide-btn' ] ifFalse: [ ^ '' ].
```

Comment:
```suggestion
	^ self accepted ifTrue: [ 'hide-btn' ] ifFalse: [ '' ]
```

Reply:
When both branches of an if do a return, we like to extract the return in Pharo
This is a detail but it makes the code more readable. The original code also works

Reply:
Thanks a lot for your reviews. I'll commit the changes you have proposed.